.taskcluster.yml

```diff
@@ -91,21 +81,23 @@ tasks:
             - tc-treeherder.v2.reference-browser.${head_rev}
           payload:
             maxRunTime: 600   # Decision should remain fast enough to schedule a handful of tasks
-            image: mozillamobile/android-components:1.15
+            image: mozillamobile/android-components:1.15 # TODO Stop using the android-components docker image.
             command:
               - /bin/bash
               - --login
               - -cx
               # The rest of the command must be defined below
             env:
-              TASK_ID: ${decision_task_id}
-              TASKS_PRIORITY: ${tasks_priority}
-              SCHEDULER_ID: ${scheduler_id}
-              BUILD_WORKER_TYPE: ${build_worker_type}
-              MOBILE_HEAD_REPOSITORY: ${repository}
+              BUILD_DATE: ${now}
```

Comment:
That's a good point. There are 3 reasons why we first implemented environment variables in mobile:

1. mozilla-central uses them quite much. That said, m-c also uses arguments.
2. All `MOBILE_` env variables are required by Chain of Trust to rebuild the context and ensure the decision task comes from a trusted repo
3. It's easier to factor them in `.taskcluster.yml` because `payload.command` is hard to factorize. That said, we can still define a variable called `$arguments` at the top of `.taskcluster.yml` and use it within each `payload.command`.

To sum up, I'm fine moving them to CLI args, we still need to expose the `MOBILE_` vars, but we don't need to use them in `decision.py`, for sure.

Reply:
As always, I appreciate how you're aware of potential repurcussions, like how CoT uses `MOBILE_`! Good point :D

Beyond this PR, this probably still won't be relevant to accomplish before taskgraph-ification, but I'm glad we had the discussion :+1: